docs/docs/reference/changed-features/implicit-resolution.md

```diff
@@ -7,39 +7,39 @@ Implicit resolution uses a new algorithm which caches implicit results
 more aggressively for performance. There are also some changes that
 affect implicits on the language level.
 
- 1. Types of implicit values and result types of implicit methods
+ 1. Types of `given` values and result types of `given` methods
```

Comment:
No, here we have to keep `implicit`. These changes apply to old-style implicit vals, not new givens. So most changes in this file need to be reverted. I'll only mark those we can keep.